server/src/test/java/org/opensearch/action/admin/cluster/remotestore/stats/TransportRemoteStoreStatsActionTests.java

```diff
@@ -90,7 +90,7 @@ public void setUp() throws Exception {
             Collections.emptySet()
         );
 
-        when(pressureService.getRemoteRefreshSegmentTracker(any())).thenReturn(mock(RemoteSegmentTransferTracker.class));
+        when(pressureService.getRemoteSegmentTransferTracker(any())).thenReturn(mock(RemoteSegmentTransferTracker.class));
```

Comment:
lets change the test to use the tracker factory.

Reply:
Updated. Resolving this thread. Please feel free to re-open if needed.